src/ray/common/client_connection.h

```diff
@@ -47,6 +66,32 @@ class ServerConnection : public std::enable_shared_from_this<ServerConnection<T>
   /// \return Status.
   ray::Status WriteMessage(int64_t type, int64_t length, const uint8_t *message);
 
+  /// Write a message to the client.
+  ///
+  /// \param type The Ray flatbuffer protocol message type.
+  /// \param length The size in bytes of the message.
+  /// \param message A pointer to the message buffer.
+  /// \return Status.
+  ray::Status WriteMessage(MessageType type, int64_t length, const uint8_t *message) {
```

Comment:
How about we make length=0 and message=nullptr the default here, so we can remove them from the call sites below?